core/arch/riscv/kernel/entry.S

```diff
@@ -0,0 +1,11 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2022 NXP
+ */
+
+#include <platform_config.h>
+#include <asm.S>
+
+FUNC _start , :
+	j	.
+END_FUNC _start
```

Comment:
I don't think it is a good idea to introduce code that is completely changed in the same series. This whole commit should be omitted.